quadcontrol/control.h

```diff
@@ -26,7 +26,8 @@ typedef enum { flight_takeoff = 0, flight_pos, flight_hover, flight_landing, fli
 
 
 typedef struct {
-	int32_t alt; /* altitude in 1E-3 [m] (millimetres) above MSL */
+	int32_t alt;     /* altitude in 1E-3 [m] (millimetres) above MSL */
+	float spoolTime; /* time of engine spool up in milliseconds */
```

Comment:
My suggestion to use shorter names.

```suggestion
	float time; /* time of motors spool up in milliseconds */
```

If you decide to change this variable, please update `static int quad_takeoff(const flight_mode_t *mode)` as well.